matter_persistence/sql/utils.py

```diff
@@ -0,0 +1,124 @@
+import logging
+from typing import Any
+
+import sqlalchemy as sa
+from sqlalchemy.orm import Query, joinedload
+
+from matter_persistence.sql.decorators import retry_if_failed
+from matter_persistence.sql.exceptions import DatabaseInvalidSortFieldError, DatabaseNoEngineSetError
+from matter_persistence.sql.models import BaseDBModel, SortMethodModel
+from matter_persistence.sql.sessions import AsyncSession, DatabaseSessionManager
+
+
+async def is_database_alive(database_session_manager: DatabaseSessionManager) -> bool:
```

Comment:
Please review this module's contents against https://sqlalchemy-utils.readthedocs.io/en/latest/database_helpers.html - maybe there are some duplicates that can be removed from here.